ledger-tool/Cargo.toml

```diff
@@ -35,6 +35,7 @@ solana-storage-bigtable = { path = "../storage-bigtable", version = "=1.7.0" }
 solana-transaction-status = { path = "../transaction-status", version = "=1.7.0" }
 solana-version = { path = "../version", version = "=1.7.0" }
 solana-vote-program = { path = "../programs/vote", version = "=1.7.0" }
+solana-core = { path = "../core", version = "=1.7.0" }
```

Comment:
Ugh, it hurts to see this dependency come into ledger-tool.  Oh well, we need to refactor core/ more at some point anyway I guess

Reply:
Ah, see your point, `core` should be internal and unexposed. Maybe make sense to move `cost model` from `core` to other better suited crates, I am not sure where though, maybe `sdk` or `runtime`?